src/pages/Home.js

```diff
@@ -1,119 +1,103 @@
+import { useState, useEffect } from 'react'
+import { Link } from 'react-router-dom'
+import { Navbar, Footer, Loading } from '../components'
+import _ from 'lodash'
+import firebase from '../data/firebase'
+import logo from '../static/oasis-logo.png'
+import '../style/App.css'
+import { colours } from '../lib/constants.js'
+import BackToTop from '../components/BackToTop'
+import { useTranslation } from 'react-i18next'
 
-import { useState, useEffect } from "react";
-import { Link } from "react-router-dom";
-import { Navbar, Footer, Loading } from "../components";
-import tools from "../data/tools.json";
-import _ from "lodash";
-import firebase from "../data/firebase";
-import logo from "../static/oasis-logo.png";
-import "../style/App.css";
-import { filterToolsByCategory } from "../utils/filterTools";
-import { colours } from "../lib/constants.js";
-import BackToTop from "../components/BackToTop";
-import { useTranslation } from "react-i18next";
+const Button = ({ category, isActive, onClick }) => {
+  return (
+    <button
+      className={`filter-button ${isActive ? 'filter-active' : ''}`}
+      title={category}
+      onClick={() => onClick(category)}
```

Comment:
All these changes are made because of prettier. We should make a prettier in all the source code to prevent those pull requests full of style updates

Reply:
Agreed!